ssm_jax/abstractions.py

```diff
@@ -217,22 +158,3 @@ def _loss_fn(params, minibatch_emissions, **minibatch_covariates):
         self.unconstrained_params = params
         return losses
```

Comment:
Good call. I’m not quite done with this refactor yet FYI. Will mark as “ready for review” and ping you on slack when it is… this will be a pretty big change though!